bigchaindb/web/views/base.py

```diff
@@ -28,4 +28,6 @@ def base_url():
 
 def base_ws_uri():
     """Base websocket uri."""
-    return 'ws://{host}:{port}'.format(**config['wsserver'])
+    # TODO Revisit as this is a workaround.
```

Comment:
I'd link to https://github.com/bigchaindb/bigchaindb/issues/1465, so if someone stumbles upon the comment they can read the description of the original issue.